docs/content/querying/topnquery.md

```diff
@@ -48,7 +48,7 @@ A topN query object looks like:
   "postAggregations": [
     {
       "type": "arithmetic",
-      "name": "sample_divide",
+      "name": "average",
```

Comment:
In the sample response it says average:
`{
        "dim1": "dim1_val5",
        "count": 60,
        "some_metrics": 2787,
        "average": 46.45
      }`